pad_server/src/main.c

```diff
@@ -96,6 +107,10 @@ int main(int argc, char **argv) {
             break;
         }
     }
+    
+    if (is_running_in_docker()){
+        setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
What does this do? I'm unfamiliar with this function.

I would prefer if we can find a way to run the simulator without knowing we're inside docker. At the very least a special command line flag would be a better solution than checking access to a file, but this is fine for now!

Reply:
Interesting, I wonder if there's a way to line buffer inside docker without changing the program itself. If so, that would be the optimal solution. For now though this is totally fine!